app/controllers/api/base_controller/parser.rb

```diff
@@ -72,7 +72,7 @@ def validate_api_parameters
       def parse_href(href)
         if href
           path = href.match(/^http/) ? URI.parse(href).path.sub(/\/*$/, '') : href
-          path = "#{@prefix}/#{path}" unless path.match(@prefix)
+          path = "/api/#{path}" unless path.match("/api")
```

Comment:
@abellotti see @Fryguy 's comment above: https://github.com/ManageIQ/manageiq/pull/11085#discussion_r78878167 - LMK if you still want to do that